### DISCUSSION TITLE
Enable the MacOS app sandbox

### DISCUSSION THREAD

**Discussion by hectour3** [mullvad/mullvadvpn-app#9069] [Feature requests & Ideas]
It would be great if Mullvad would opt into using the MacOS application sandbox. ProtonVPN uses it too, and personally for me it is the only thing preventing me from switching. On first sight it does not seem like that much work to me.

https://developer.apple.com/documentation/security/app_sandbox
https://developer.apple.com/documentation/xcode/configuring-the-macos-app-sandbox

**Comment by raksooo** [comment]
Thanks for the suggestion! We've added this to our backlog and will eventually look into it.

**Comment by securitybrahh** [comment]
What, this is a huge issue!

Should pin it.🤷
